TranslationAssistant.Business/DocumentTranslationManager.cs

```diff
@@ -37,6 +37,24 @@ namespace TranslationAssistant.Business
 
     #endregion
 
+    public class DocumentTranslatorResult
+    {
+        private int sourceFileCharCount = 0;
+        private int targetFileCharCount = 0;
+
+        public int SourceFileCharCount
```

Comment:
These can just be `public int SourceFileCharCount { get; set; }` as the 0 is the default value